.github/workflows/ci.yml

```diff
@@ -47,7 +47,15 @@ jobs:
         uses: ./.github/actions/unit-test-setup
 
       - name: Run unit tests
-        run: ./mvnw install -nsu -B -DskipTestsuite -DskipQuarkus -DskipExamples
+        run: |
+          SEP=""
+          PROJECTS=""
+          for i in `find -name '*Test.java' -type f | egrep -v './(testsuite|quarkus)/' | sed 's|/src/test/java/.*||' | sort | uniq | sed 's|./||'`; do
+            PROJECTS="$PROJECTS$SEP$i"
+            SEP=","
+          done
+          
+          ./mvnw test -nsu -B -pl "$PROJECTS" -am
```

Comment:
This results in a slightly faster execution as only the modules (and what they depend on) with tests are built, and not the whole project.